src/main/java/seedu/eylah/commons/core/index/Index.java

```diff
@@ -15,7 +15,7 @@
      * Index can only be created by calling {@link Index#fromZeroBased(int)} or
      * {@link Index#fromOneBased(int)}.
      */
-    private Index(int zeroBasedIndex) {
+    public Index(int zeroBasedIndex) {
```

Comment:
Why change the private to public for the Index class?

Reply:
I had to create a Index object for some of my test cases.

Reply:
okay, thanks for the clarification